integrations/server_test.go

```diff
@@ -293,7 +294,8 @@ func TestServer(t *testing.T) {
       "default": true,
       "telegraf": "telegraf",
       "organization": "howdy",
-      "defaultRP": "",
+			"defaultRP": "",
+			"authentication": "basic",
```

Comment:
ditto ^

Reply:
These are json strings and this is the result of gofmt